src/main/java/com/gregtechceu/gtceu/common/data/GTMachines.java

```diff
@@ -187,7 +189,8 @@ public class GTMachines {
                     .recipeModifier(SteamBoilerMachine::recipeModifier)
                     .workableSteamHullRenderer(pressure, GTCEu.id("block/generators/boiler/solar"))
                     .tooltips(Component.translatable("gtceu.universal.tooltip.produces_fluid",
-                            (pressure ? 360 : 120) * FluidHelper.getBucket() / 20000))
+                                    (pressure ? ConfigHolder.INSTANCE.machines.smallBoilers.hpSolarBoilerBaseOutput :
+                                    ConfigHolder.INSTANCE.machines.smallBoilers.solarBoilerBaseOutput) * FluidHelper.getBucket() / 1000))
```

Comment:
Base output is in steam per second, the tooltip is in steam per tick, it makes sense for it to be divided by 20

Reply:
Ahhh makes sense